resources/assets/components/InboxItem/index.js

```diff
@@ -52,12 +70,12 @@ class InboxItem extends React.Component {
           </ul>
         </div>
         <div className="container__block -third">
-          {post['user'] ?
+          {post['user'] || signup.user ?
             <div>
-              <h2>{post['user']['first_name']} {post['user']['last_name']}, {calculateAge(post['user']['birthdate'])}</h2>
+              <h2>{first_name} {last_name}, {birthdate}</h2>
```

Comment:
ohh I like this idea! Especially since we were talking about how it's a little weird to have a space between first name and comma if there is no last name. Would I have to make a call to gateway to get this?